pettingzoo/utils/wrappers/pz_to_gymnasium_wrapper.py

```diff
@@ -0,0 +1,78 @@
+from typing import Callable, Dict, Optional, Protocol, TypeVar, Union
+import gymnasium
+from pettingzoo import AECEnv
+
+
+Action = TypeVar("Action")
```

Comment:
Good to think about things in this way but I would import the actual ActionType which we define in env.py, just for consistency and if we end up changing the types in the future it is better to reference them.